deployments/fuji/usdc/migrations/1665028496_absorb_transfer_event.ts

```diff
@@ -0,0 +1,62 @@
+import { DeploymentManager, migration } from '../../../../plugins/deployment_manager';
+import { calldata, exp, proposal } from '../../../../src/deploy';
```

Comment:
nit: looks like `calldata`, `exp`, `expect` are all unused. same with the `COMP` on line 21

Reply:
I guess we don't lint the `/deployments/` directory. I'll create a PR to enable that

Reply:
#601